stl/inc/array

```diff
@@ -434,6 +434,9 @@ struct pointer_traits<_Array_iterator<_Ty, _Size>> {
 template <class _Ty, size_t _Size>
 class array { // fixed size array of values
 public:
+    static_assert(!is_reference_v<_Ty>, "The C++ Standard forbids containers of reference elements "
+                                        "because of [container.requirements].");

```

Comment:
Would it make sense to make this a macro?

Reply:
I believe so

Reply:
You might have to see the maintainers' opinion here.

Reply:
We try to avoid macros in new code, unless something is extremely repetitive that can't be dealt with in another way, or it's platform-dependent and needs to vary based on external conditions.